var/spack/repos/builtin/packages/py-bluepyemodel/package.py

```diff
@@ -10,11 +10,12 @@ class PyBluepyemodel(PythonPackage):
     """Python library to optimize and evaluate electrical models."""
 
     homepage = "https://github.com/BlueBrain/BluePyEModel"
-    pypi = "bluepyemodel/bluepyemodel-0.0.57.tar.gz"
+    pypi = "bluepyemodel/bluepyemodel-0.0.46.tar.gz"
 
-    version("0.0.57", sha256="0b91e39e5066ab4a996bd932577b49648169e549c5f05bb3f93e345b4b186093")
+    version("0.0.46", sha256="ad4c125e491f3337fcc341a4f389b8a616d883ce50fd77d9fb0ea6e13be5da61")
```

Comment:
Why the downgrade?

Reply:
The newer version was merged without upstreaming and I didn't notice :-(